File: app/src/main/java/me/eagzzycsl/darkroom/db/AboutDb.java
package me.eagzzycsl.darkroom.db;


interface AboutDb {
    String DB_NAME = "DarkRoom";
    int DB_VERSION = 1;

    interface TABLE_Naughty {
        String TABLE_NAME = "naughty";

        interface FILED {
            String ID = "_ID";
            String pkgName = "pkgName";
        }
    }
}

File: app/src/main/java/me/eagzzycsl/darkroom/db/AboutDb.kt
package me.eagzzycsl.darkroom.db


object AboutDb {

    object NaughtyTable {

        object FILED {
            const val ID = "_ID"
            const val pkgName = "pkgName"
            const val isNaughty = "isNaughty"
            const val freezeNotifyTimes = "freezeNotifyTimes"
        }

        const val TABLE_NAME = "naughty"
    }

    const val DB_NAME = "DarkRoom"
    const val DB_VERSION = 2
}
